scijava/scijava-ops-api/templates/main/java/org/scijava/ops/api/OpBuilder.vm

```diff
@@ -433,28 +433,38 @@ public class OpBuilder {
 			this.outType = outType;
 		}
 
+		/**
+		 * Match a {@link org.scijava.function.Functions} op to produce an instance of this builder's {@code outType}.
+		 *
+		 * @see #computer
```

Comment:
Why is there a link to `#computer` here? For contrast with `function`? I think you can add a description after the `@see #computer` blurb explaining *why* the reader should see it, which could be helpful here.